pyproject.toml

```diff
@@ -62,9 +62,9 @@ dependencies = [
   "mmh3",  # fast hashing function (murmurhash3)
   "quantulum3",  # quantities extraction from text
   "posthog",  # telemetry
-  "azure-ai-formrecognizer==3.2.0b2",  # forms reader
+  "azure-ai-formrecognizer>=3.2.0b2",  # forms reader
```

Comment:
Just to make sure, have you actually tried to install a later version of this package and run tests and tutorials with that? I have a vague memory that we pinned this beta release for a good reason, even though I can't recall why right now. @bogdankostic do you know more about this?

Reply:
@ZanSara When the code has been last updated, the azure-ai-formrecognizer latest version available was b2. The other versions would not have the DocumentAnalysisClient available. 

pip would not force a beta over a GA. So, without the pin, 3.1.2 was being chosen. The new versions all have DocumentAnalysisClient available and some improvements (support for new API version). Despise MS released latest as GA, pip has yet the b6 tag there.